actions/ci-lint-go/action.yml

```diff
@@ -103,7 +103,7 @@ runs:
 
     - name: Set env from files
       if: inputs.use-env-files == 'true'
-      uses: cardinalby/export-env-action@66657b34899a2d695434ed060d9f2215db9b4035 # v2.2.0
+      uses: cardinalby/export-env-action@04460b5708ae685cda6419c29deb8903ca2db74a # 2.2.1
```

Comment:
@erikburt any idea why it removed the `v` in this update?

Reply:
There's no `v` on the tag - https://github.com/cardinalby/export-env-action/releases/tag/2.2.1

Reply:
this one has no tag either:
https://github.com/cardinalby/export-env-action/releases/tag/2.2.0
yet we currently have `v2.2.0`?

Reply:
It was probably set incorrectly initially - https://github.com/smartcontractkit/.github-archive/blame/09b5e05a727a912be5f02a8c4449081f8047a349/actions/ci-lint-go/action.yml#L106